### ISSUE TITLE
Slack migration from public to Stanford instance

### ISSUE THREAD

**Issue by jonc101** [HealthRex/CDSS#205]
Main difference is apparently permanent archiving (vs. public Slack deletes messages >6 months old or something).

**Comment by JonathanChiang** [comment]
https://uit.stanford.edu/service/slack/migration

**Comment by jonc101** [comment]
Non urgent, but yes, can you look into making this migration happen?
From what I understand, the main benefit is permanent thread history, which may be important if someone needs to go back and trace a conversation.

Jonathan

**Comment by JonathanChiang** [comment]
yes, i can look into it.

**Comment by JonathanChiang** [comment]
Extremely important! **NOTES on premigration** 
Users MUST have their default email address entered in their Slack workspace account.  The default email is the one that is set as “default” in accounts.stanford.edu and displays when viewing their information in stanfordwho.stanford.edu.

If a user is using an alias or non-default email address when migrated, the account created on Stanford Slack Grid will not be attached to their history in the workspace.  This can be fixed by the Org admins without loss of history, but it will delay the user’s ability to access your workspace.

It is highly recommended that you communicate this need ahead of time to your team and/or audit your user accounts prior to migration.  

Single-Channel Guests and the Stanford Slack Grid
Many workspaces use Single-Channel Guests to have members from other workgroups participate in their workspaces. In Stanford Slack Grid a user can only have one role across the Grid. They are either a full member of the Grid or a single channel guest on the Grid.

If you currently have Single-Channel Guests in your workspace and they are already a full member account on the Grid, they will be automatically deactivated from your workspace after migration. You will be able to direct message them regardless. (In some cases there are issues in direct messaging former Single-Channel Guests. You can fix that issue using the troubleshooting steps below.) If you still want them to participate in a channel you will need to share that channel with the user's home workspace.

If there are a large number of Single-Channel Guests using your workspace with you, consider creating a new workspace for that combined workgroup.

**Comment by JonathanChiang** [comment]
*submitted form for premigration to stanford IT*

**Comment by jonc101** [comment]
I don’t really know what all this means. Can you send out a succinct message to the group for any changes they need to be aware of or plan for? Thanks.

**Comment by JonathanChiang** [comment]
- [x] Setup Migration Time for May 2nd 
- [ ] Close ticket once migration is complete

**Comment by JonathanChiang** [comment]
Migration complete: Stanford Grid: may have to make new issue on user account linking
